cloudservice/aws/aws_test.go

```diff
@@ -26,22 +26,22 @@ var _ = Describe("AWS Parsing", func() {
 
 		It("should parse the ip range file", func() {
 			By("Ensuring that no error was thrown")
-			Expect(err).To(BeNil())
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
Just want to know, what's the difference between those two?

Reply:
There is no difference in reality.
And now there is a Ginkgo linter to help you know it's not the right one you are using 😉 

It's basically more syntaxically correct and readable "out loud" to say "I expect this error to not have occurred".

This is the way 😄